documentation/guide/OWSGuide.adoc

```diff
@@ -116,7 +129,73 @@ It is also possible to specify the content of the whitelist in the response file
 
 #<TODO: describe OWS settings options>#
 
-=== Centralized Configuration
+=== System Configuration
+
+When loading the configuration during the start of OpenWebStart the following steps are executed:
+
+1. Load the default values which are hardcoded in the source code.
+1. Search for a system configuration.
+1. Load the system configuration if one was found.
+1. Load the user configuration.
+
+Whenever a configuration is loaded the values which are already defined are updated.
+There is however the possibility to lock a property.
+If a property is locked then subsequent configurations may not modify the value.
+This allows enforcing certain values on a system level.
+Any changes the user makes in his personal configuration file will not have any effect on the locked property.
+
+==== Defining the System Configuration
+
+The system configuration needs to be defined in the following way.
+
+*Windows:* create the file `%windir%\Sun\Java\deployment\deployment.config` and add the following properties:
+
+*MacOs and Linux:* create the file `/etc/.java/deployment/deployment.config` and add the following properties:
+
+deployment.system.config:: The URL to the system configuration.
+Special characters need escaping.
+See the following examples:
+* `deployment.system.config=file\:/C\:/Window/Sun/Java/system.properties`
```

Comment:
did you change the name on purpose to system.properties to distinguish from deployment.properties? Maybe we should mention that the name is not predefined but is taken as specified here?